File: Day-9/program 1.py
problem statement :
    
python program to get digit root of number where we give a random number and until it become complete single digit it goes on calculate
    
solution :
        
num = input()
def droot(num):
    if len(num) == 1:
        return num
    else:
        sum = 0
        for i in num:
            sum += int(i)
        num = str(sum)
        return droot(num)
print("The digital root of ", num, "is: ",        
droot(num))


output :
    
16
The digital root of 16 is:7

File: Day-9/program 1.c
#include <iostream> 
using namespace std; 

int find_value(int a[], int n, int k) 
{ 
    
    int sum = 0; 
  
   
    for (int i = 0; i < n; i++) { 
        sum += a[i]; 
    } 
 
    return sum % k; 
} 
  

int main() 
{ 
    int n = 5, k = 3; 
    int a[] = { 12, 4, 13, 0, 5 }; 
    cout << find_value(a, n, k); 
    return 0; 